Source/Core/Core/HW/EXI/BBA/TAP_Win32.cpp

```diff
@@ -31,6 +31,7 @@ bool IsTAPDevice(const TCHAR* guid)
     HKEY unit_key;
     TCHAR component_id_string[] = _T("ComponentId");
     TCHAR component_id[256];
+    TCHAR* component_id_sub;
```

Comment:
This can be declared within the if-statement that uses it.

Reply:
That was my initial idea, but seeing everything defined at the top level of each function scope, I figured it was part of the style.

Reply:
I declared it within the if-statement now.